### PR TITLE
Allow ohm-cmd to generate recipe for ProtoBuiltInRules; useful for bootstrapping.

### PR DESCRIPTION
**Pull request by tonyg**

### CODE REVIEW

File: src/Grammar.js [hunk 1]
```diff
@@ -169,7 +164,7 @@ Grammar.prototype = {
       var body = self.ruleBodies[ruleName];
 
       var operation;
-      if (self.superGrammar.ruleBodies[ruleName]) {
+      if (self.superGrammar && self.superGrammar.ruleBodies[ruleName]) {
```
Comment (alexwarth) [inline src/Grammar.js]: Would you mind adding a `isExtendingOrOverriding` variable that holds the value of this condition, and then...

File: src/Grammar.js [hunk 2]
```diff
@@ -182,7 +177,8 @@ Grammar.prototype = {
       }
 
       var description = null;
-      if (!self.superGrammar.ruleBodies[ruleName] && self.ruleDescriptions[ruleName]) {
+      if (!(self.superGrammar && self.superGrammar.ruleBodies[ruleName]) &&
```
Comment (alexwarth) [inline src/Grammar.js]: ... referencing it down here, i.e., `!overridingOrExtending && ...`? I think that would make this code a little more readable. Thanks!

**Comment by alexwarth** [comment]
Thanks, Tony! Looks great. Will merge after you make the little change I requested.

**Comment by tonyg** [comment]
I messed that up. Sorry. I'll open a fresh PR.

**Pull request merged**